src/main/java/Deadline.java

```diff
@@ -0,0 +1,27 @@
+public class Deadline extends Task{
+
+    protected String by;
+
+    public Deadline(String description){
+        super(description);
+        //this.by = by;
+    }
+
+    public void setBy(String description){
+        String[] arrOfStr = description.split("/", 2);
```

Comment:
Consider spelling out arrOfStr fully as it may vague to arrayOfString to make code more readable.
Similarly for the other variables in the other classes, would be good if you can spell them out fully.